org-roam-node.el

```diff
@@ -180,7 +180,7 @@ populated."
                                     (magit-section-up)
                                     (org-roam-node-at-point)))
         (t (org-with-wide-buffer
-            (org-back-to-heading-or-point-min)
+            (org-back-to-heading-or-point-min t)
```

Comment:
Good catch. There are other places where this needs to be fixed as well:

https://github.com/org-roam/org-roam/blob/82733e133ab03b0138201e28dcb06e073a4fb3cc/org-roam-node.el#L868
https://github.com/org-roam/org-roam/blob/82733e133ab03b0138201e28dcb06e073a4fb3cc/org-roam-node.el#L830

basically all calls to `org-back-to-heading-or-point-min`.